lib/fog/ovirt/requests/compute/list_disk_profiles.rb

```diff
@@ -0,0 +1,20 @@
+module Fog
+  module Compute
+    class Ovirt
+      class Real
+        def list_disk_profiles filter={}
+          client.diskprofiles(filter)
+        end
+      end
+
+      class Mock
+        def list_disk_profiles(filters = {})
+          xml = read_xml 'list_disk_profiles.xml'
```

Comment:
@freshfrmthehood I think that abenari/rbovirt#47 is just waiting for you to rebase your changes and squash. Could you take a look at it and then over here so we can get this merged?

Thanks!